levechad/settings.py

```diff
@@ -171,10 +171,6 @@
 MEDIA_ROOT = os.path.join(BASE_DIR, 'mediafiles')
 
 if ENV == 'PRODUCTION':
-    # Enforcing TLS/SLL
-    SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
-    SECURE_SSL_REDIRECT = True
-
     # In production, use S3 to serve static & media files
```

Comment:
Will this break the current VPS?

Reply:
no, we don't use this settings.py in the vps